crates/neon-runtime/src/napi/error.rs

```diff
@@ -1,11 +1,24 @@
-use raw::Local;
+use raw::{Env, Local};
 
-pub unsafe extern "C" fn throw(_val: Local) { unimplemented!() }
+use nodejs_sys as napi;
 
-pub unsafe extern "C" fn new_error(_out: &mut Local, _msg: Local) { unimplemented!() }
+pub unsafe extern "C" fn throw(env:Env, error: Local)->bool {
+    let status=napi::napi_throw(env,error);
```

Comment:
If you could reformat this to match the style of the rest of Neon that would make for better consistency. There's a lot of missing whitespace, for example surrounding punctuation like `=` and `,` and `->` and operators like `==`.